examples/3d/chess.rs

```diff
@@ -0,0 +1,482 @@
+            parent.spawn_bundle(PbrBundle {
+                mesh: bishop_handle.clone(),
+                material: black_material.clone(),
+                transform: {
+                    let mut transform = Transform::from_translation(Vec3::new(-1.5, 0., -3.0));
+                    transform.apply_non_uniform_scale(Vec3::new(0.18, 0.18, 0.18));
+                    transform.rotate(Quat::from_rotation_x(1.55));
+                    transform.rotate(Quat::from_rotation_y(1.35));
+                    transform
+                },
+                ..Default::default()
+            });
+        })
+        .id();
+    commands
+        .spawn_bundle(PbrBundle {
+            transform: Transform::from_translation(Vec3::new(0., 0., 0.)),
+            ..Default::default()
+        })
+        .with_children(|parent| {
+            parent.spawn_bundle(PbrBundle {
+                mesh: bishop_handle.clone(),
+                material: black_material.clone(),
+                transform: {
+                    let mut transform = Transform::from_translation(Vec3::new(1.5, 0., -3.0));
+                    transform.apply_non_uniform_scale(Vec3::new(0.18, 0.18, 0.18));
+                    transform.rotate(Quat::from_rotation_x(1.55));
+                    transform.rotate(Quat::from_rotation_y(1.35));
+                    transform
+                },
+                ..Default::default()
+            });
+        })
+        .id();
+    commands
+        .spawn_bundle(PbrBundle {
+            transform: Transform::from_translation(Vec3::new(0., 0., 0.)),
+            ..Default::default()
+        })
+        .with_children(|parent| {
+            parent.spawn_bundle(PbrBundle {
+                mesh: knight_handle.clone(),
+                material: black_material.clone(),
+                transform: {
+                    let mut transform = Transform::from_translation(Vec3::new(-2.5, 0., -3.0));
+                    transform.apply_non_uniform_scale(Vec3::new(0.18, 0.18, 0.18));
+                    transform.rotate(Quat::from_rotation_x(1.55));
+                    transform.rotate(Quat::from_rotation_y(0.));
+                    transform
+                },
+                ..Default::default()
+            });
+        })
+        .id();
+    commands
+        .spawn_bundle(PbrBundle {
+            transform: Transform::from_translation(Vec3::new(0., 0., 0.)),
+            ..Default::default()
+        })
+        .with_children(|parent| {
+            parent.spawn_bundle(PbrBundle {
+                mesh: knight_handle.clone(),
+                material: black_material.clone(),
+                transform: {
+                    let mut transform = Transform::from_translation(Vec3::new(2.5, 0., -3.0));
+                    transform.apply_non_uniform_scale(Vec3::new(0.18, 0.18, 0.18));
+                    transform.rotate(Quat::from_rotation_x(1.55));
+                    transform.rotate(Quat::from_rotation_y(0.));
```

Comment:
The scaling and rotation is identical for every piece per-color - except black knights. These values should probably be constants, or passed into the create piece functions

Reply:
Went ahead and created constants and aligned the orientation of Black Knights to match the rest of the team to reduce complexity.